tcb/.vscode/tasks.json

```diff
@@ -2,8 +2,9 @@
     "version": "2.0.0",
     "tasks": [
         {
-            "label": ".🔃-try-update-template",
-            "command": "echo",
+            "label": "try-update-template",
+            "detail": "This task has automation steps to update the project\nbased on the template used to create it.",
```

Comment:
Shouldn't the \n be removed here also?

Reply:
no, the break lines are added the most close possible to 80 and 120, because without then the tooltip will be a long line.